### PR TITLE
UCL 552 Allow DataGrid consumers to easily map values to keys to filter data with keys/IDs

### PR DESCRIPTION
**Pull request by lubos-tomsa**

### CODE REVIEW

**Review by szmejk — APPROVED** [review]

File: src/components/DataGrid/DataGridFilters/FilterKeyMapper.test.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      expect(m.columnKvPairs.get("col1")).toEqual(kvPairs1);
+      expect(m.columnKvPairs.get("col2")).toEqual(kvPairs2);
+    });
+
+    it("should provide values", () => {
+      expect(m.getValues("col1")).toEqual(["v1", "v2", "v3"]);
+      expect(m.getValues("col2")).toEqual(["V1", "V2", "V3"]);
+    });
+
+    it("should provide values", () => {
```
Comment (szmejk) [inline src/components/DataGrid/DataGridFilters/FilterKeyMapper.test.ts:57]: should provide keys for values?

File: src/components/DataGrid/DataGridFilters/FilterKeyMapper.test.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      expect(m.getKeysForValues("col1", ["v1", "v3"])).toEqual(["k1", "k3"]);
+      expect(m.getKeysForValues("col2", ["V1", "V3"])).toEqual(["K1", "K3"]);
+    });
+
+    describe("error scenarios", () => {
+      it("should raise error on wrong column for getValues", () => {
```
Comment (szmejk) [inline src/components/DataGrid/DataGridFilters/FilterKeyMapper.test.ts:63]: for getKeyPairs

**Review by szmejk — CHANGES_REQUESTED** [review]

File: src/components/DataGrid/DataGridFilters/FilterKeyMapper.test.ts (hunk 1 above)
Comment (lubos-tomsa) [reply src/components/DataGrid/DataGridFilters/FilterKeyMapper.test.ts]: fixed

File: src/components/DataGrid/DataGridFilters/FilterKeyMapper.test.ts (hunk 2 above)
Comment (lubos-tomsa) [reply src/components/DataGrid/DataGridFilters/FilterKeyMapper.test.ts]: fixed

**Review by szmejk — APPROVED** [review]

**Pull request merged**